pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/RegexpExtractTransformFunction.java

```diff
@@ -81,10 +81,10 @@ public void init(List<TransformFunction> arguments, Map<String, ColumnContext> c
     }
 
     if (arguments.size() == 4) {
-      TransformFunction positionFunction = arguments.get(3);
-      Preconditions.checkState(positionFunction instanceof LiteralTransformFunction,
+      TransformFunction defaultValueTransformFunction = arguments.get(3);
+      Preconditions.checkState(defaultValueTransformFunction instanceof LiteralTransformFunction,
           "`default_value` must be a literal expression.");
-      _defaultValue = ((LiteralTransformFunction) regexpFunction).getStringLiteral();
+      _defaultValue = ((LiteralTransformFunction) defaultValueTransformFunction).getStringLiteral();
     } else {
```

Comment:
Isn't this just a rename in the variable?

Reply:
Oh! Forget what I said, I didn't see we are using a different variable in line 87